home-clients/pom.xml

```diff
@@ -72,8 +72,6 @@
                             <generateApiTests>false</generateApiTests>
                             <generateModelTests>false</generateModelTests>
                             <configOptions>
-                                <java8>true</java8>
-                                <dateLibrary>java8</dateLibrary>
```

Comment:
Why did you remove it? Can it affect anything?

Reply:
DateLibrary got back, because it needs for jdk8+.

Reply:
No. I already did.